haystack/components/converters/__init__.py

```diff
@@ -33,6 +34,7 @@
     from .json import JSONConverter
     from .markdown import MarkdownToDocument
     from .msg import MSGToDocument
+    from .multi_file_converter import MultiFileConverter
```

Comment:
let's also update [pydoc config](https://github.com/deepset-ai/haystack/tree/main/docs/pydoc/config) to populate the API reference